tokio-macros/src/lib.rs

```diff
@@ -28,6 +28,11 @@ use proc_macro::TokenStream;
 /// without requiring the user to use [Runtime](../tokio/runtime/struct.Runtime.html) or
 /// [Builder](../tokio/runtime/struct.builder.html) directly.
 ///
+/// Note: This macro is designed to be simplistic and targets applications that do not require
+/// a complex setup. If provided functionality is not sufficient, user may be interested in
+/// using [Builder](../tokio/runtime/struct.builder.html), which provides a more powerful
```

Comment:
```suggestion
/// using [Builder](../tokio/runtime/struct.Builder.html), which provides a more powerful
```
And similarly for the other link.